tuf/client/client.go

```diff
@@ -204,8 +204,12 @@ func (c Client) verifyRoot(role string, s *data.Signed, minVersion int) error {
 	// Still need to determine if there has been a root key update and
```

Comment:
We should update the comment in the line above this one, which references `c.keysDB`